### ISSUE TITLE
Image file name too long

### ISSUE THREAD

**Issue by emahadzimustafic** [emahadzimustafic/type-those-grids#1]
Hi! I keep getting this error in GitHub and I haven't been able to create any commits:

error: lstat("images/Maerten_van_Heemskerck_-_Panorama_with_the_Abduction_of_Helen_Amidst_the_Wonders_of_the_Ancient_World_-_Walters_37656.jpg"): Filename too long
fatal: Unable to process path images/Maerten_van_Heemskerck_-_Panorama_with_the_Abduction_of_Helen_Amidst_the_Wonders_of_the_Ancient_World_-_Walters_37656.jpg

Do I have to rename the image file? 

@hellachella @carolsinc

**Comment by carolsinc** [comment]
All of your images must follow naming conventions so you will need to rename the files, and make sure you change them accordingly in your code

**Comment by emahadzimustafic** [comment]
Ohh okay. I renamed all the image files and fixed their names in the img tags, but my **seven-wonders.jpg** image isn't showing up on my website. 

Prior to all of this, when I forked the repository, I didn't have this image file in my image folder. I had to download it separately (from Github website) and put it into the folder. Would that be the reason why it's not working?

@hellachella @carolsinc
